src/app/js/lib/getTitle.js

```diff
@@ -1,13 +1,26 @@
 import { getCleanHost } from '../../../common/uris';
 
+const HOST_REGEX = /https?:\/\/([\w-]+)/;
+
 export default () => {
     const host = getCleanHost();
+
     /**
-     * @type {string}
+     * @type {string[]}
      */
-    const tenant = window.__TENANT__ ?? 'default';
+    const title = [];
+
+    if (window && window.__TENANT__) {
+        title.push(window.__TENANT__);
+    } else {
+        title.push('default');
+    }
+
+    if (host) {
+        title.push(HOST_REGEX.exec(host)[1]);
+    } else {
+        title.push('example');
+    }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why display `example` when you don't have the host?
```suggestion
```

Reply:
Don't no, this was this way before.